Sabbath School/Read/Controller/ReadController.swift

```diff
@@ -603,10 +632,15 @@ extension ReadController: ReadViewOutputProtocol {
     }
 
     func didReceiveHighlights(readHighlights: ReadHighlights) {
+        if Helper.isPad {
+            highlights.append(readHighlights)
+        }
+        
         presenter?.interactor?.saveHighlights(highlights: readHighlights)
     }
 
     func didReceiveComment(readComments: ReadComments) {
+        updateCommentsListWhenRotateIpad(readComments: readComments)
```

Comment:
@EmersonCarpes Are you sure we need this? These two funcs are triggered when comments or highlights are being saved (e.g after highlighting) and subsequently saved to DB.

Reply:
@VitalikL I did this way to solve the scenario when is reading lesson, highlight something and rotate device, otherwise would be necessary call cache/api every time that user rotates device